Flutter/calendar/builders.md

```diff
@@ -252,6 +252,11 @@ class MyAppState extends State<MyApp> {
 >**NOTE**
 * For recurring appointments, this will always return as `Appointment`, even for the custom business objects.
 
+### How to identify the spanned appointment view date in Flutter Calendar?
+
+By using the [appointmentBuilder](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/SfCalendar/appointmentBuilder.html) date property we can identify the appointment view start date. For example in month view we have spanned appointment(StartTime - August 17,2022 and EndTime - August 30,2022) and the appointment renders with 3 view, the appointmentBuilder holds the start date (August 20,2022, August 21,2022 and August 28,2022) of the each appointment view.  Please [click here]()to view the sample.
```

Comment:
Using the [appointmentBuilder](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/SfCalendar/appointmentBuilder.html) date property, we can identify the appointment view start date. For example, in month view, we have spanned appointments (StartTime - August 17, 2022, and EndTime - August 30, 2022), and the appointment renders with 3 views. The appointmentBuilder holds the start date (August 20, 2022, August 21, 2022, and August 28, 2022) of each appointment view.  Please [click here]()to view the sample.